website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -372,13 +368,13 @@ spec:
 Create the CR:
 
 ```sh
-$ kubectl apply -f config/samples/cache_v1alpha1_memcached.yaml
+$ kubectl apply -f config/samples/cache_v1alpha1_memcached.yaml -n memcached-operator-system
```

Comment:
It isn't necessary to create the sample in the same directory as the operator since the operator is watching and reconciling CRs in all namespaces.

I would prefer that we not explicitly specify namespace for `kubectl` commands related to the CR and its resources.

I think this means that starting with this command, and all kubectl commmands below should be reverted.

Reply:
So, if I go that @joelanford's suggestion here the goal would show that the CR can be applied in any ns. In this case, in the default one and getting reconciled since the operator is cluster scope by default. 
+1

Reply:
alright I have amended the PR accordingly